### DISCUSSION TITLE
live duration top right on the stream preview

### DISCUSSION THREAD

**Discussion by degu-Tanya** [night/betterttv#6038] [Ideas]
![image](https://user-images.githubusercontent.com/100243423/227859732-cc2f60f1-9143-452f-90c6-912b7944a0e2.png)
live duration top right on the stream preview, as a option
![image](https://user-images.githubusercontent.com/100243423/227863049-db54043f-07a2-4ca8-9617-26dbfc5130e3.png)

**Comment by qs-arno** [comment]
I'm a long-time FFZ user who's using BTTV now and thinking of switching...this is one thing I immediately miss.  Would be great to have the duration in the "followed channels" list that goes down the right side when you're on an individual channel page, too.
